tools/inundate_gms.py

```diff
@@ -131,122 +147,209 @@ def Inundate_gms( hydrofabric_dir, forecast, num_workers=1,
-    parser.add_argument('-f','--forecast',help='Forecast discharges in CMS as CSV file',required=True)
-    parser.add_argument('-i','--inundation-raster',help='Inundation Raster output. Only writes if designated.',required=False,default=None)
-    parser.add_argument('-p','--inundation-polygon',help='Inundation polygon output. Only writes if designated.',required=False,default=None)
-    parser.add_argument('-d','--depths-raster',help='Depths raster output. Only writes if designated. Appends HUC code in batch mode.',required=False,default=None)
-    parser.add_argument('-l','--log-file',help='Log-file to store level-path exceptions',required=False,default=None)
-    parser.add_argument('-o','--output-fileNames',help='Output CSV file with filenames for inundation rasters, inundation polygons, and depth rasters',required=False,default=None)
-    parser.add_argument('-w','--num-workers', help='Number of Workers', required=False,default=1)
-    parser.add_argument('-v','--verbose',help='Verbose printing',required=False,default=None,action='store_true')
-    
\ No newline at end of file
+    parser = argparse.ArgumentParser(description="Inundate FIM")
+    parser.add_argument(
+        "-y",
+        "--hydrofabric_dir",
+        help="Directory path to FIM hydrofabric by processing unit",
+        required=True,
+    )
+    parser.add_argument(
+        "-u",
+        "--hucs",
+        help="List of HUCS to run",
+        required=False,
+        default=None,
+        type=str,
+        nargs="+",
+    )
+    parser.add_argument(
+        "-f", "--forecast", help="Forecast discharges in CMS as CSV file", required=True
+    )
+    parser.add_argument(
+        "-i",
+        "--inundation-raster",
+        help="Inundation Raster output. Only writes if designated.",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "-p",
+        "--inundation-polygon",
+        help="Inundation polygon output. Only writes if designated.",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "-d",
+        "--depths-raster",
+        help="Depths raster output. Only writes if designated. Appends HUC code in batch mode.",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "-l",
+        "--log-file",
+        help="Log-file to store level-path exceptions",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "-o",
+        "--output-fileNames",
+        help="Output CSV file with filenames for inundation rasters, inundation polygons, and depth rasters",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "-w", "--num-workers", help="Number of Workers", required=False, default=1
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        help="Verbose printing",
+        required=False,
+        default=None,
+        action="store_true",
+    )
```

Comment:
It appears as though PR #948 removed the call to `Inundate_gms(**vars(parser.parse_args()))`. Not sure if the removal of the function call was intentional or not, but as this stands, a call to this script does not execute any code. I would propose to either:

1. add the call to `Inundate_gms()` back so that executing this script does indeed run the function
2. remove the ability to call this script directly (remove `argparse` & `if __name__ == "__main__":` )

Reply:
Good catch. I think it was accidental and we do want to keep it so I've added it back into this PR.